src/ToricVarieties/LineBundles/attributes.jl

```diff
@@ -21,3 +22,36 @@ function variety(l::ToricLineBundle)
     return l.variety
 end
 export variety
+
+
+@doc Markdown.doc"""
+    toric_divisor(l::ToricLineBundle)
+
+Returns a divisor corresponding to the toric line bundle `l`.
+"""
+function toric_divisor(l::ToricLineBundle)
+    return get_attribute!(l, :toric_divisor) do
+        class = divisor_class(l)
+        map1 = map_from_cartier_divisor_group_to_picard_group(variety(l))
+        map2 = map = map_from_cartier_divisor_group_to_torus_invariant_divisor_group(variety(l))
+        image = map2(preimage(map1, class)).coeff
+        coeffs = vec([Int(x) for x in image])
+        td = ToricDivisor(variety(l), coeffs)
+        set_attribute!(td, :iscartier, true)
+        return td
+    end
+end
+export divisor
+
+
+@doc Markdown.doc"""
+    degree(l::ToricLineBundle)
+
+Returns the degree of the toric line bundle `l`.
+"""
+function degree(l::ToricLineBundle)
+    return get_attribute!(l, :degree) do
+        return sum(coeffs(toric_divisor(l)))::Int
```

Comment:
I guess this also should be `fmpz`?

Reply:
Actually since Julia should be able to infer that `coeffs(toric_divisor(l))` is a `Vector{Int}` (and perhaps soon a `Vector{fmpz}` ?) it should also infer the return type here correctly.